apps/snmp_exporter/app.yaml

```diff
@@ -0,0 +1,71 @@
+{
+    "id": "/snmp_exporter",
+    "cmd": null,
+    "cpus": 1,
+    "mem": 1024,
```

Comment:
is this much RAM necessary?

Reply:
Probably not, but I'm not sure how to measure the proper amount. Any suggestions?

Reply:
htop inside container?